eggplant/membership/tests.py

```diff
@@ -162,7 +158,7 @@ def test_send_invitation(self):
         expected = 'Invitation has been send to {}'.format(invited_email)
         self.assertContains(response, expected, 1, 200)
 
-        self.assertEqual(len(mail.outbox), 1)
+        self.assertEqual(len(mail.outbox), 1)  # @UndefinedVariable
```

Comment:
It is still hideous and clutters up code 😝 Also, this is a test, it will break if `.outbox` doesn't exists :)